src/pages/MapViewer/components/Toolbox/components/ToolboxItems/Preview/styles.scss

```diff
@@ -0,0 +1,7 @@
+.preview {
```

Comment:
The size of the button is slightly less than the other buttons, and the colour is different. It looks like the colour of the other icons are a dark grey and this one is black